src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -205,6 +223,18 @@ protected void writeToNBT(NBTTagCompound compound)
         }
         compound.setTag(TAG_CITIZENS, citizenTagList);
 
+        //  Achievements
+        NBTTagList achievementsTagList = new NBTTagList();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'achievementsTagList' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)